ArduPlane/quadplane.cpp

```diff
@@ -3861,7 +3862,7 @@ float QuadPlane::get_weathervane_yaw_rate_cds(void)
     float wv_output;
     if (weathervane->get_yaw_out(wv_output,
                                      plane.channel_rudder->get_control_in(),
-                                     plane.relative_ground_altitude(plane.g.rangefinder_landing),
+                                     plane.relative_ground_altitude(RangeFinderUse::TAKEOFF_LANDING),
```

Comment:
Just a note that weathervane is not only use on take off and landing. It also works in QLoiter. This is a change in default behavior.

Reply:
yes, though if you don't want it for takeoff/landing you are unlikely to want it for loiter. It's a trade off between complexity and usefulness